examples/text_to_image/train_text_to_image.py

```diff
@@ -163,33 +166,57 @@ def log_validation(vae, text_encoder, tokenizer, unet, args, accelerator, weight
     else:
         generator = torch.Generator(device=accelerator.device).manual_seed(args.seed)
 
-    images = []
-    for i in range(len(args.validation_prompts)):
-        with torch.autocast("cuda"):
-            image = pipeline(args.validation_prompts[i], num_inference_steps=20, generator=generator).images[0]
+    validation_prompts = [
+        "cute Elon Musk character",
+        "A pokemon with wings",
+        "a photo of yoda",
+        "a cute creature with blue wings",
+    ]
```

Comment:
Shouldn't be like this.

Reply:
I should take from arguments, right? `args.validation_prompts`